.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+sudo: required
+jdk: oraclejdk8
+before_script:
+  - sudo apt-get update && sudo apt-get install oracle-java8-installer
+script:
+  - npm test
+  - npm run danger
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - node_modules
+env:
+  global:
+    - SAUCE_USERNAME=cerner
+    - secure: xbS6srAElIfUiioKUO8IQzGoFhwgI0VXu7yXjMixlJ8FX3pdXCH3w67HtpgdfW/9Uy04N2UmhH6vpmPmuE0geZI/tuMavLYpokvqVWg0SQuA1im28uBuIia65lxrQTrHOxCvrfNT5kjSvuuZ9bASJseh4HpmKbDXz2FszslDstZVvS/pOeO/7qKzS2pgQVO4SLsDveAODzqeUZEllQlsyPjuY90KBSWvxhJ6cB+IglK/8yS/FJMkVFTnFrlAWv10oD9wNiQvOfnn/y0+UjSyFiCJogOoRxAGNPZQ8q6QIWNgquEfL+/Lg0N2B+MHNwimQXg4Nk4yT4nyg7j+wT8eFANnOl9Q84sY1QxMRxtn6CpFNEHX+2zfAsdUEKdhm/mHJxxhEMJcFjlK1FqpJGF/eV5fYXRt2semrk/72l+kApCUPwgohR43pN2B9Fmwnb4ScEL5VaOcsZOeTUlLlZacjcWeW8CpvhEdTU9H1uRzKyqAsnzfEKT+qAontgbvGyuduzykTRuWgR5T4F1NiT0vjJ1XGkX3RoX4mZG0HnagU7Kg3CAoNLvveCPO12m5mHFSeG3W4WD0lI01fICh4dHsipHkHQwTHMw3nd4MoxH4erRqSDfoatfkgY0iNN8rgfCH/5iNgg5jwnyLz4MymgwZt4NzpQGjvDTxP442XHY1AVg=
+before_deploy:
+  - lerna run --scope terra-consumer-site compile:prod
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
```

Comment:
You'll need to setup this token in travis CI to get the site to auto deploy to GH pages on the master build.
https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings